README.md

```diff
@@ -87,8 +88,9 @@ X = pd.read_csv('data/MNIST-1k.csv', sep=' ', header=None).to_numpy()
 X_tsne = TSNE(n_components = 2).fit_transform(X)
 
 # Fit the data with BanditPAM:
-kmed = KMedoids(n_medoids = 10, algorithm = "BanditPAM")
-kmed.fit(X, 'L2', 10, "mnist_log")
+k = 10
```

Comment:
ordinarily it's good practice to define variables and avoid magic numbers, but in this case for the README examples, since `k` is only used once, let's not use a separate variable and just pass `10` in the call below